package.json

```diff
@@ -23,13 +23,13 @@
     "author": "Emmanuelle Gouillart <emmanuelle.gouillart@nsup.org>",
     "license": "MIT",
     "dependencies": {
-        "ramda": "^0.25.0",
-        "react": "15.4.2",
-        "react-dom": "15.4.2",
-        "react-sketch": ">=0.4.4",
         "material-ui": "^0.20.0",
-        "react-color": "^2.13.8",
-        "plotly-icons": ">=1.0"
+        "plotly-icons": ">=1.0",
+        "ramda": "^0.25.0",
```

Comment:
I was going to suggest updating ramda, but it doesn't seem to be used here at all, can we just 🔪 ?

Reply:
https://github.com/plotly/dash-canvas/pull/47/commits/dd843afd410e60eabb7f59d93bddb71a191315b5